question3/question3.c

```diff
@@ -0,0 +1,195 @@
+#include <stdio.h>
```

Comment:
question 3 readme wasn't updated -5
the output also doesn't seem quite right -5
10/20

Feel free to try some more on this question